bad_exclusions_list.py

```diff
@@ -0,0 +1,112 @@
+bad_exclusions_list = [
+    "THIS IS NOT A LIST OF EXCLUSIONS IN YOUR ENVIRONMENT, BUT A LIST OF EXCLUSIONS WE RECOMMEND AGAINST!",
+    "For the most up to date list and additional information refer to",
+    "https://www.cisco.com/c/en/us/support/docs/security/amp-endpoints/213681-best-practices-for-amp-for-endpoint-excl.html",
+    "",
```

Comment:
empty exclusion shouldn't be possible, no?  also are 2-4 supposed to be comments?

Reply:
Moved these comments to separate elements like they should have been from the start.
<img width="589" alt="image" src="https://user-images.githubusercontent.com/42283335/220999849-b104dd43-18b0-493a-b88b-9e2943df83c8.png">